docusaurus/docs/reference/tunnelers/linux/linux-tunnel-options.md

```diff
@@ -69,3 +69,78 @@ If the DNS record exists it returns the answer and sets query status to `NO_ERRO
 #### System Requirements for Mode `run-host`
 
 `ziti-edge-tunnel run-host` does not require elevated privileges or the above device or socket, only network egress to the servers for which it is hosting Services.
+
+## Multi-Factor Authentication
+
+You may use multi-factor authentication (MFA) with `ziti-edge-tunnel` by completing the following workflow to register a time-based one-time password (TOTP) token generator with the OpenZiti Controller.
+
+### Find the Identifier of the OpenZiti Identity
+
+The identifier is typically the full path to the JSON identity file that is already added to the running `ziti-edge-tunnel run` process.
+
+The `ziti-edge-tunnel tunnel_status` command will list the identifiers of all identities that are currently loaded. The user that runs this command, and all the `ziti-edge-tunnel` commands that follow, must have write permission on the IPC socket, i.e. `/tmp/.ziti/ziti-edge-tunnel.sock`.
+
+Optionally, you may use `jq` to parse the JSON output of `ziti-edge-tunnel tunnel_status` to extract the identifiers. I found it necessary to also truncate the output to remove the leading and trailing characters around the JSON.
```

Comment:
Don't lead with adjectives like this. We need to just tell the user what needs to be done, or tell them where to find the right data. this whole line should be removed